modules/KalturaSupport/resources/mw.kAnalony.js

```diff
@@ -400,7 +400,8 @@
 
 			this.embedPlayer.bindHelper('playerError', function (e, errorObj) {
                 var errorCode = errorObj && errorObj.key;
```

Comment:
easier to change this to:
```suggestion
                var errorCode = errorObj && errorObj.code;
```
no?

Reply:
as mentioned in description - trying to keep backward compatibility and didn't want to go everywhere where we set  key and change to code as it might be used by external dependencies